packages/ozone/tests/lang.test.ts

```diff
@@ -15,7 +15,7 @@ describe('moderation status language tagging', () => {
 
   beforeAll(async () => {
     network = await TestNetwork.create({
-      dbPostgresSchema: 'ozone_blob_divert_test',
```

Comment:
dang I think I had this fixed in one of my inprogress prs, prolly the mod-manager one? anyways, thanks for the fix!